packages/manager/apps/telecom/src/app/telecom/telephony/line/assist/troubleshooting/translations/Messages_fr_FR.json

```diff
@@ -33,7 +33,7 @@
   "telephony_line_assist_troubleshooting_problem_phone_mac_address": "Adresse MAC : {{ macAddress }}",
   "telephony_line_assist_troubleshooting_support_contact": "Contacter le support",
   "telephony_line_assist_troubleshooting_support_contact_title": "Votre problème semble nécessiter l'intervention d'un technicien.",
-  "telephony_line_assist_troubleshooting_support_contact_way_to": "Veuillez contacter le support OVH&#8239;:",
+  "telephony_line_assist_troubleshooting_support_contact_way_to": "Veuillez contacter le support OVHcloud:",
```

Comment:
i think we should preserve the Non-breaking space (&#8239;)

Reply:
done ty